src/core/model/LineStyle.h

```diff
@@ -61,4 +70,9 @@ class LineStyle: public Serializable {
      * Dash count (0 for no dash)
      */
     int dashCount = 0;
+
+    /**
+     * Flag to clear LineStyle selection
+     */
+    bool clearLineStyle = false;
```

Comment:
I don't like this: the flag has nothing to do with the LineStyle itself, so it should not be part of the data structure.

I guess the way I'd do this is by adding a tool capability (see ToolEnums.h::ToolCapabilities and Tool::setCapability)

Reply:
I agree.

I decided to re-write the fix using a new tool capability as you suggested.
In my opinion, the new fix makes way more sense than the previous workaround and does not introduce code where it doesn't belong (e.g. in `XournalView::setSelection()`, `ToolHandler::setSelectionEditTools())`).